contributingGuides/KSv2.md

```diff
@@ -0,0 +1,67 @@
+# KSv2
+
+In Expensify, we use a prioritization system called called "Kernel Scheduler v2" (KSv2) to help us keep aligned on how frequently GitHub issues should be worked on, depending on their time priority:
+
+**Hourly**: If a Github issue has the `Hourly` label, the assignee should update it *at least* once an hour
+
+**Daily**: If a Github issue has the `Daily` label, the assignee should update it *at least* once a day
+
+**Weekly**: If a Github issue has the `Weekly` label, the assignee should update it *at least* once a week
+
+**Monthly**: If a Github issue has the `Monthly` label, the assignee should update it *at least* once a month.
+
+## Dashboard
+
+To help surface the issues and PRs that need the most `#urgency`, we've built a custom extension to use in GitHub. It looks like this:
+
+<img src="https://user-images.githubusercontent.com/6829422/213875977-8ff4cf19-7690-4203-ae13-a8da259be7d0.png" />
+
+Once you have the extension installed, you can access it by going to https://github.com/Expensify/Expensify#k2 or clicking on the K2 tab at the top of a repo (between pull requests and GH actions).
+
+### Pull Requests for review
+
+In the dashboard, you can first see the PRs assigned to you as `Reviewer`. As part of our engineering guidelines, an engineer (internal or external) should review other people's code before working on their own code.
```

Comment:
```suggestion
In the dashboard, you can first see the PRs assigned to you as `Reviewer`. As part of our engineering guidelines, an engineer (internal or external) should review other people's code before working on their own code. This helps unblock others so that everyone can be as efficient as possible getting stuff done.
```